community-roadmap/backend/README.md

```diff
@@ -0,0 +1,60 @@
+# Community Roadmap for Zendesk Help Center
+
+This project contains a service that maps Zendesk community posts to product roadmap items. The service is built as a Cloudflare Worker with a Cloudflare KV data store.
```

Comment:
Let's assume the user read the top-level README. You can link to it if you're worried about them just landing here. Let's focus on what the service does and how it relates to the backend.

```suggestion
The service is built as a Cloudflare Worker with a Cloudflare KV data store.
```

I'd remove the first sentence, but I think we need some more context about what this backend service does. Can we provide links to the Cloudflare Worker docs for users who may not be familiar with it? Does this project require Cloudflare to work?